src/lib/BadgeProfile.js

```diff
@@ -200,11 +202,67 @@ export default function BadgeProfile({ ev, ...rest }) {
       authors: [ev.pubkey],
     },
   });
+
+  const following = useNostrEvents({
+    filter: {
+      kinds: [CONTACT_LIST],
+      authors: [user],
+    },
+  });
+
   const name = findTag(ev.tags, "name");
   const description = findTag(ev.tags, "description");
   const image = findTag(ev.tags, "image");
   const naddr = encodeNaddr(ev);
   //const thumb = findTag(ev.tags, "thumb");
+
+  async function followPubKeys(awardEvents) {
+    const followEvent = {
+      kind: CONTACT_LIST,
+      tags: [],
+      created_at: dateToUnix(),
+      content: "",
```

Comment:
Can we keep the content from the old event? in practice clients are putting a JSON string with relay metadata there and don't want to nuke it.

Reply:
makes sense, i'll revert

Reply:
done